### DISCUSSION TITLE
Alpine Not Loading if show_progress_bar is set to false in config

### DISCUSSION THREAD

**Discussion by faakher** [livewire/livewire#8716] [7. Bugs (archived)]
### Livewire version

v3.5.4

### Laravel version

v11.19.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Set show_progress_bar to false in livewire.config file with inject_assets set to true

Although this issue is fixed in previous version as described in this PR [#7158](https://github.com/livewire/livewire/pull/7158) 
There is data-no-progress-bar parameter in livewire.js but still facing issue

### Repository or Wirebox URL

No response

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [ ] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by BorisKM** [comment]
Keep it with `true` value.
Just hide `nprogress` in CSS:
```css
#nprogress {
    @apply hidden;
}
```
